packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -464,6 +502,17 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
           } else if (pendingOperation != null && pendingOperation.method.equals(METHOD_INIT)) {
             finishWithError(ERROR_REASON_CONNECTION_FAILED, String.valueOf(resultCode));
           }
+          return true;
+        } else if (requestCode == REQUEST_CODE_RECOVER_AUTH) {
+          if (resultCode == Activity.RESULT_OK) {
+            getTokens(pendingOperation.result, (String) pendingOperation.data, false);
```

Comment:
Do we know the pendingOperation type?

I would assert that pendingOperation was getTokens().